package excecao;

public class ChecadaVsNChegada {

    public static void main(String[] args) {

	try {
	    geraErro1();
	} catch (RuntimeException e) {
	    System.out.println(e.getMessage());
	    e.printStackTrace();
	}

	geraErro2();
	
	
	System.out.println("Fim ?");
    }

    // NÃ£o checada
    static void geraErro1() {
	throw new RuntimeException("Ocorreu um erro #01");
    }

    static void geraErro2() {
	new Exception("Ocorreu um erro #02");
    }
}
